public-api/src/main/java/org/pmiops/workbench/utils/mappers/QuestionConceptMapper.java

```diff
@@ -0,0 +1,34 @@
+package org.pmiops.workbench.cdr;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.cdr.model.DbQuestionConcept;
+import org.pmiops.workbench.model.QuestionConcept;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(
+        config = MapStructConfig.class,
+        uses = {CommonMappers.class})
+public interface QuestionConceptMapper {
+    @Mapping(target = "conceptId", source = "db.dbQuestionConceptId.conceptId")
+    @Mapping(target = "conceptName", source = "db.conceptName")
```

Comment:
mappings are not required when properties have the same name.. only when they don't match. I would remove all line that fit that pattern.